src/components/time-picker/TimePicker.tsx

```diff
@@ -434,4 +415,13 @@ const handleFocus = (event) => {
   }
 };
 
+TimePicker.defaultProps = {
+  disabledTimes: [],
+  format: getUserFormat(),
+  min: '00:00:00',
+  max: '23:59:59',
+  step: STEP.DEFAULT_STEP_VALUE,
```

Comment:
is step an enum?

Reply:
Yes it is

Reply:
The default value is taken from an enum, but it is a number in seconds. (min: 600, default: 900, max 43200). The step value can be anything between min and max